tools/initializer/src/main/java/org/sa/rainbow/initializer/configuration/ConfigurationLoader.java

```diff
@@ -0,0 +1,73 @@
+package org.sa.rainbow.initializer.configuration;
+import org.sa.rainbow.initializer.models.Variable;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Map.Entry;
+import java.util.*;
+import java.io.*;
+
+/**
+ * The Configuration class represent a set of key-values of required variables
+ */
+
+public class ConfigurationLoader {
+    /**
+     * A map of key values for the required variables.
+     */
+    private Map<String, String> config;
+    /**
+     * A list of variables required in the entire Model
+     */
+    private List<Variable> localVariables;
+    private Set<String> variableSet;
+
+    public ConfigurationLoader(){
+        config = new HashMap<String, String>();
```

Comment:
Use diamond operator whenever possible.
```suggestion
        config = new HashMap<>();
```